contracts/rollup/BatchManager.sol

```diff
@@ -39,6 +39,7 @@ contract BatchManager {
     event NewBatch(uint256 batchID, bytes32 accountRoot, Types.Usage batchType);
     event StakeWithdraw(address committed, uint256 batchID);
     event RollbackStatus(uint256 startID, uint256 nDeleted, bool completed);
+    event RollbackTriggered(uint256 batchID, Types.Result result);
```

Comment:
This is a comment, no action is required.

We have two events that will be emitted in the startRollingBack, let me explain why we need two. The short answer is RollbackTriggered is only called once per dispute and the RollbackStatus is called once per rollback.

For example, we have a long rollback that takes too much gas so we need to slash in multiple steps. Say the tip batch is 120, the disputed batch is 100, and then the last good batch is 99.

- The first step we submit the fraud-proof and determine if we need to do startRollingBack. The fraud-proof part might already takes a lot of gas. The startRollingBack is initiated and then it emits RollbackTriggered, which tells us 100 is disputed and determined invalid for BadSignature reason. startRollingBack then calls rollback
- this rollback() only rolls back from 120 to 110 (110 not included), which emits RollbackStatus telling us it "started" from batchID 120, "deleted" n = 10 batches, and not "completed" yet.
- some other people called keepRollingBack. It now rolls back from 110 to 100. RollbackStatus tells us this time it "started" from batchID 110, "deleted" n = 10 batches, and "completed" the slash.